concepts/Common/Security Considerations/40 Content Security Policy/00 Content Security Policy.md

```diff
@@ -1,4 +1,4 @@
-Content Security Policy (CSP) is a security feature that helps to detect and mitigate certain types of attacks, such as clickjacking, cross-site scripting (XSS), and other malicious code injection attacks. These attacks allow hackers to perform everything from data theft to page spoofing, malware distribution, or site defacement. It is important to ensure that components of DevExtreme UI library are compatible with CSP.
+[Content Security Policy](https://developer.mozilla.org/ru/docs/Web/HTTP/CSP) (CSP) is a security feature that helps to detect and mitigate certain types of attacks, such as clickjacking, cross-site scripting (XSS), and other malicious code injection attacks. These attacks allow hackers to perform everything from data theft to page spoofing, malware distribution, or site defacement. It is important to ensure that components of DevExtreme UI library are compatible with CSP.
```

Comment:
Please wrap the link in <a> tag, not a markdown link, so it could be opened in a new tab (add "target: _blank")

Reply:
It opens in a new tab. I have checked its behavior.